### ISSUE TITLE
Play/Run button for the currently open .collection file in the Editor.

### ISSUE THREAD

**Issue by selimanac** [defold/defold#9166]
Since we can bootstrap any `.collection` file by setting the `Main Collection` at game.project, it would be nice to have an  button to build the currently open .collection file on Editor. Similar to Unity's or Godot's Play buttons.

**Comment by JCash** [comment]
We wouldn't want to have this as a default behavior, as it would be cumbersome having to select different tabs (away from your current collection), in order to launch the game as we currently do.

We do have the option to add a bootstrap collection when booting.
```
"--config=bootstrap.main_collection=/gui_render_crash/gui_render_crash.collectionc"
```

Currently, I'm not sure if our editor script would allow for adding that flag just-in-time when the user presses "Build and run", but I'd rather see this done via editor extensions.

**Comment by AGulev** [comment]
this old PR seems related https://github.com/defold/defold/pull/1910/files

**Comment by britzl** [comment]
> this old PR seems related https://github.com/defold/defold/pull/1910/files

Yes, that's exactly the one that was meant to solve the problem. Not sure why it wasn't completed.

**Comment by AGulev** [comment]
> Yes, that's exactly the one that was meant to solve the problem. Not sure why it wasn't completed.

I think this was a bit tricky approach. I would go with an approach Mathias suggested: 
* regular build and run but
* with "--config=bootstrap.main_collection=/path_to/custom_sollectiom.collectionc"
